src/services/VerifyAccountRequestProcessor.ts

```diff
@@ -147,7 +148,7 @@ export class VerifyAccountRequestProcessor {
   		restricted: {
   			Experian_request_details: [
```

Comment:
Is this up to date? It doesnt match whats in the request catalogue

Reply:
Updated